css/sass/_header.scss

```diff
@@ -129,23 +129,20 @@ header {
       @media (min-width: 480px) {
         margin: 0 auto;
       }
+      @media (max-width: 480px) {
+          margin-right: auto;
+          margin-left: auto;
+          max-width: 83%;
+      }
     }
   }
-  .search-expander {
-    height: 2.1em;
+  .search-expander, .search-close {
+    height: 2.7em;
```

Comment:
Hi @punalsc - why the change to `.search-expander` height? Has something changed?